pom.xml

```diff
@@ -109,6 +111,52 @@
                     <scmCommentPrefix xml:space="preserve">MOJITO RELEASE: </scmCommentPrefix>
                 </configuration>
             </plugin>
+
+            <plugin>
+                <groupId>com.diffplug.spotless</groupId>
+                <artifactId>spotless-maven-plugin</artifactId>
+                <version>2.24.1</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>apply</goal>
+                        </goals>
+                        <phase>compile</phase>
+                    </execution>
+                </executions>
+                <configuration>
+                    <pom>
+                        <sortPom>
+                            <nrOfIndentSpace>4</nrOfIndentSpace>
+                        </sortPom>
+                    </pom>
+                    <java>
+                        <googleJavaFormat>
+                            <version>${plugin.version.google-java-format}</version>
+                            <style>GOOGLE</style>
+                        </googleJavaFormat>
+                        <importOrder>
+                            <wildcardsLast>false</wildcardsLast>
```

Comment:
if we have auto formatting so we avoid wildcards? those in some cases it is nice to have for static imports.

Reply:
Yes - it's to avoid wildcard imports. I'm generally not too bothered about wildcard imports - but I wanted to align it with what we tend to do in the project currently.

I haven't seen any setting that can be used to differentiate between the context of where that's used, either on/off everywhere. It's easiest to stick to the style guide in this instance (no wildcard imports).

According to the Google Java style guide, wildcard imports are not recommended for static imports either:
https://google.github.io/styleguide/javaguide.html
`3.3.1 No wildcard imports
Wildcard imports, static or otherwise, are not used.`

Reply:
Nope - I mis-read what the setting does - it's just about ordering - will remove it.

Reply:
from what I've read in the past it is better to avoid them. One case i though was handy though with `l10nmessages` was to be able to wildcard the message ids, so you don't have to bother to get something to compile as you're editing the properties file. that's a very specific use case though

Reply:
Just to confirm: that setting didn't control the actual wildcard part - only where wildcards are placed within the list.
We'll still have:
`import static org.junit.Assert.*;`